README.md

```diff
@@ -13,14 +13,13 @@ Download the latest `acli.phar` file from the [releases](https://github.com/acqu
 
 For example, you can run following simple script:
 ```bash
-curl -OL https://github.com/acquia/cli/releases/download/v1.0.0-rc3/acli.phar
-mv acli.phar acli
-chmod +x acli
+curl -OL https://github.com/acquia/cli/releases/download/v1.0.0-rc4/acli.phar
+chmod +x acli.phar
 ```
 
 Next, place it somewhere globally accessible on your machine. For instance:
 ```
-mv acli /usr/local/bin/
+mv acli.phar /usr/local/bin/acli
```

Comment:
reduced in one line the steps, but that's maybe a bit pedantic from me

Reply:
Why remove the permissions change?

Reply:
it's still there, line 17 (chmod +x acli.phar), but you are saving one line of code doing the move from acli.phar to acli when moving it to the final place.

Again, a bit pedantic, so I can leave it as it was before